src/macro-external/twitch/macro-action-twitch.cpp

```diff
@@ -131,8 +175,10 @@ bool MacroActionTwitch::Save(obs_data_t *obj) const
 	obs_data_set_int(obj, "action", static_cast<int>(_action));
 	obs_data_set_string(obj, "token",
 			    GetWeakTwitchTokenName(_token).c_str());
-	_text.Save(obj, "text");
+	_streamTitle.Save(obj, "streamTitle");
```

Comment:
Just to mention it:
This will break your existing macros.

Reply:
Yeah, changed it as that was not released yet and your recommendation when I asked, so hopefully won't be a problem (no idea myself :P)